tests/acceptance/features/cliDataExporter/export.feature

```diff
@@ -74,3 +74,12 @@ Feature: An administrator wants to export the files of his user using
     When user "unknown" is exported to path "/tmp/fooSomething" using the occ command
     Then the command should have failed with exit code 1
     And the command output should contain the text "Could not extract user metadata for user"
+
+  @issue-209
+  Scenario: exporting an user with empty trash bin by deleting trashbin's items
+    Given user "Alice" has uploaded file with content "hello" to "testfile1.txt"
+    And user "Alice" has uploaded file with content "hello world" to "testfile2.txt"
+    And user "Alice" has deleted file "/testfile1.txt"
+    And user "Alice" has emptied the trashbin
+    When user "Alice" is exported to path "/tmp/fooSomething" using the occ command
```

Comment:
may be add a step for the success of the occ command in `Then` step

Reply:
ok